lua/nvim-tree/legacy.lua

```diff
@@ -190,6 +190,27 @@ local function refactored(opts)
       end
     end
   end
+
```

Comment:
These are ugly.

I started to write a generic function to do this, but it was more ugly.

Reply:
could you refactor the 3 line nested if statement into one in `refactored` ? Doesn't look right :smile:

Reply:
Collapsed all.